kubeflow/jupyter/NOTEBOOK.MD

```diff
@@ -0,0 +1,121 @@
+## Creating a Jupyter notebook in user-defined namespaces
+In order to create a notebook in a given namespace, Kubeflow expects a service account named `jupyter-notebook`, whcih allows K8s resource access,  and a secret named `user-gcp-sa,` which allows GCP service access, to exist in the namespace. 
```

Comment:
nit; "whcih" -> "which"

Can we move the instructions that are GCP specific into their own section and clarify that they only apply on GCP?